packages/package_info/example/integration_test/package_info_test.dart

```diff
@@ -2,6 +2,8 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+// @dart = 2.9
+
 import 'dart:io';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:integration_test/integration_test.dart';
```

Comment:
@dnfield I couldn't find an issue about migrating `vm_service` to null safe. Do you know if there's any work in the radar?

Reply:
Looks like per https://github.com/dart-lang/sdk/issues/40046 it's been landed. We can look to migrate integration_test and driver.

Reply:
Awesome!

Reply:
I was wrong. That's for a different part of the SDK. Migration is still ongoing, Dart cl is https://dart-review.googlesource.com/c/sdk/+/156980